dbConnectionFunction.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+
+/**Function creates a connection to the database
+ *
+ * @return PDO
+ */
+function dbConnection() {
```

Comment:
Exactly what I meant! You can also add a return type hint here like this:

function dbConnection() :PDO {}